samples/RefreshFlow/AuthorizationServer/web.config

```diff
@@ -2,8 +2,13 @@
 <configuration>
   <system.webServer>
     <handlers>
-      <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified"/>
+      <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModuleV2" resourceType="Unspecified" />
```

Comment:
We should probably remove this file (and other web.config files if there are any).